config/backend.yml

```diff
@@ -3,7 +3,7 @@ version: '3'
 services:
 
   db:
-    image: mysql:5.7
+    image: mysql:8.0
```

Comment:
I don't think we need to change mysql image in this PR

Reply:
Okay
I reverted this change